component/Card/style.css

```diff
@@ -0,0 +1,196 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  font-family:san-serif;
+  background-color: rgb(151, 151, 151);
+  font-family: sans-serif;
+  display:flex;
+  gap: 1rem;
+  margin: 1rem;
+  flex-wrap: wrap;
+}
+
+/* text only card */
+
+
+.card-background {
```

Comment:
can be renamed as card-bg(prefer shorter and easy to understand class names)

Reply:
done